frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1507,7 +1536,7 @@ const ManageHostsPage = ({
         {count !== undefined && (
           <span>{`${count} host${count === 1 ? "" : "s"}`}</span>
         )}
-        {count ? (
+        {!!count && (
```

Comment:
Renders a 0 without !!

Reply:
We can't use count !==0 because we want the count to render if it's 0, but not if it errors / undefined